### DISCUSSION TITLE
Manually run/trigger AdvancedCronJob?

### DISCUSSION THREAD

**Discussion by dev-samples** [openkruise/kruise#967] [Q&A] [answered]
A `kind: CronJob` can be manually created with:

`kubectl create job --from=cronjob/my-cronjob my-cronjob-manual-01
`
How do I manually create a `kind: AdvancedCronJob`? I have tried with:

`
kubectl create job --from=advancedcronjobs.apps.kruise.io/my-adv-job my-adv-job-manual-001
`
where:

```
kubectl get advancedcronjobs.apps.kruise.io 
NAME                     SCHEDULE       TYPE           LASTSCHEDULETIME   AGE
my-adv-job   0 02 * * SAT   BroadcastJob                      25m

```

**Comment by hantmac** [comment, marked as answer]
@dev-sample The `kubectl` does not support `advancedcronjobs.apps.kruise.io` which is not the native resource of k8s, OpenKruise has a plugin [kubectl-plugin](https://github.com/openkruise/kruise-tools), this plugin can manipulate Kruise's workload resources, so we can implement the command in this plugin.
